docs/development/testguide.rst

```diff
@@ -48,13 +48,12 @@ Running Tests
 *************
 
 There are currently three different ways to invoke Astropy tests. Each
-method invokes `pytest`_ to run the tests but offers different options when
-calling. To run the tests, you will need to make sure you have the `pytest`_
+method invokes |pytest| to run the tests but offers different options when
+calling. To run the tests, you will need to make sure you have the |pytest|
 package installed.
 
 In addition to running the Astropy tests, these methods can also be called
-so that they check Python source code for `PEP8 compliance
-<https://www.python.org/dev/peps/pep-0008/>`_. All of the PEP8 testing
+so that they check Python source code for |PEP8|. All of the PEP8 testing
 options require the `pytest-pep8 plugin
 <https://pypi.org/project/pytest-pep8>`_, which must be installed
```

Comment:
Side note: I never even installed `pytest-pep8`. The dev guide could probably use some serious clean-up but that is a battle for another day.